middlewares/validators/progresses.js

```
@@ -144,9 +152,78 @@ const validateGetDayProgressParams = async (req, res, next) => {
+            }),
+            planned: joi.string().trim().required().messages({
+              "any.required": "Required field 'planned' is missing.",
+              "string.trim": "planned must not have leading or trailing whitespace",
+            }),
+            blockers: joi.string().trim().allow("").required().messages({
+              "any.required": "Required field 'blockers' is missing.",
+              "string.trim": "blockers must not have leading or trailing whitespace",
+            }),
+            taskId: joi.string().trim().when("type", {
+              is: "task",
+              then: joi.required().messages({
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)